tests/test_auth.py

```diff
@@ -27,6 +27,38 @@ def test_custom_token_server():
     assert DummyRequest.headers['Authorization'] == 'Bearer a'
 
 
+@responses.activate
+def test_custom_token_server_custom_auth_scheme():
+    """
+    RFC 2617 standard on HTTP Authentication states that credentials should
+    consist of at least a scheme and arbitrary parameters:
+        credentials = auth-scheme #auth-param
+
+    In practice auth-scheme is often "Bearer" because it is used in the OAuth2 standard.
+    This is our default, the initial request made to the custom_token_server will be
+    used in the the Authorization header after the Bearer auth-scheme.
+
+    But we support altenative auth-schemes in this case the `filter` should emit the
+    compltete value of the Authorization header.
```

Comment:
Thanks for this docstring :100:

Reply:
typo: compltete > complete

Reply:
```suggestion
    RFC 2617 standard on HTTP Authentication states that credentials should
    consist of at least a scheme and arbitrary parameters:
        credentials = auth-scheme #auth-param

    In practice auth-scheme is often "Bearer" because it is used in the OAuth2 standard.
    This is our default, the initial request made to the custom_token_server will be
    used in the the Authorization header after the Bearer auth-scheme.

    But we support altenative auth-schemes in this case the `filter` should emit the
    complete value of the Authorization header.
```